src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.tHead.addEventListener('click', (event) => {
+  sortList(event.target.cellIndex);
+});
+
+function sortList(index) {
+  const sortedRows = [...table.tBodies[0].rows].sort((a, b) => {
+    let first = a.cells[index].textContent;
+    let second = b.cells[index].textContent;
+
+    if (index === 3) {
```

Comment:
it's better to find a universal condition to identify a necessary column (new columns can be added and this index will change)

Reply:
yes, your right. but the title text can change too. i can't think of a universal condition here for all cases

Reply:
yes, agree. but this case was too obvious to ignore ;)